source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsApiController.cs

```diff
@@ -135,7 +135,7 @@ public JsonNetResult DeletePeriod(string period)
 
         [AcceptVerbs(HttpVerbs.Post)]
         [CommonJobsAuthorize(Roles = "EvaluationManagers")]
-        public JsonNetResult ChangeResponsable (string evaluatedUserName, string period, string newResponsibleName)
+        public JsonNetResult ChangeResponsible (string evaluatedUserName, string period, string newResponsibleName)
```

Comment:
I updated the documentation and I'll help her with the URL once this change is published